apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -548,6 +548,8 @@ const EventTypePage = (props: EventTypeSetupProps) => {
       clearTimeout(timeout);
     };
   }, []);
+
+  const [disableSave, setDisableSave] = useState<boolean>(true);
```

Comment:
@moulibrota-das instead of creating a state to track form changes how about we use the watch api https://react-hook-form.com/docs/useform/watch instead?